### ISSUE TITLE
FUTURE OF THIS EMULATOR!!

### ISSUE THREAD

**Issue by ghost** [SachinVin/citra_android#209]
@SachinVin and @HelloOO7  will ever new updates and features will be added to this emulator??
I know of the **official Citra emulator**, but for the time being......
Will features like **HLE emulation** and **Installing Updates** to games will be introduced??
OR
Is this the **end??**

**Comment by Nishant765** [comment]
I know right! He hasn't released any sort of update or news in here since the past 7 months. He hasn't even replied to anybody. I think this emulator is dead.

**Comment by ghost** [comment]
But some updates should be introduced that are necessary
Like HLE AAC Decoder(for games such as Pokemon X)
Installing updates using .Cia
Limit framerate
Local Internet(not that necessary)

Some temporary builds should be released

**Comment by Dippt** [comment]
There is an official Citra android version in progress

**Comment by HelloOO7** [comment]
Well yeah, as the official version is coming along, SachinVin has decided that maintaining this repo is no longer needed. There is no ETA as to when it goes live as there are not many people working on it, but most of the changes from SachinVin's OpenGL ES backend have been merged, though targeting 3.1 instead of 3.2. The only difference is that there's no frontend.

If you want 3DS games on the go, really, just use a 3DS. The Android version is not a priority for that very reason, so it'll take a few months to years (unlike a \*cough\* certain \*cough\* YouTuber said) until it's finished. Maybe, by that time, phones will actually be able to run them through the interpreter.

For full featured Citra, play on a PC. Or be prepared to wait for a really, really long time.

**Comment by ghost** [comment]
@HelloOO7  but at least by introducing some features you will also help the **Official Citra Team** to introduce those same features in their official builds!!
Well, it's your choice..
Still this emulator can emulate some of the games at full speeds if you use a good mobile( I use **Oneplus 6T McLaren Edition**)
I think this repo should be maintained

**Comment by DzuSyimmalain** [comment]
For leeco le 2 please

**Comment by Nishant765** [comment]
@Hello007 Yeah I know that this emulator will not be released for another 2 to 3 years. The only reason I wanted this emulator was because I had a potato pc and I can't run Citra on that. But I do have a very powerful phone(One Plus 6) to run it. But this year I'm buying Alienware Area 51M so I won't be needing this Android Citra anymore . So yeah if you guys are waiting for the official release it will probably be 2020 to 2022. And this release will not have the full features as the pc version and may need another year to be fully functional. Anyway as I was saying if you guys can't wait for 3 years, just play on the pc version and if you don't have a pc, just save up money and buy it. It's better than waiting for 3 years.

**Comment by kquote03** [comment]
Btw, I think there is a working version of official citra android that can be compiled.
Just try to compile it through android studio (using official repo) 
I tried to do this but my pc can't handle it, and freezes...

**Comment by ghost** [comment]
Please support Mikage a 3Ds emulator for android
https://mikage.app/

**Comment by GunneR007** [comment]
@everyone sorry to inform use but he is no longer working on this project because citra has taking over it and from wat I hear is that the project is dead
